cosmic_ray/util.py

```diff
@@ -64,5 +64,7 @@ def build_mutations(ops, to_ops):
     return [
         (idx, to_op)
         for idx, from_op in enumerate(ops)
-        for to_op in to_ops(from_op)
+        # note: mutations to self are excluded.
+        # None is a special mutation meaning to delete the operator!
+        for to_op in to_ops(from_op) if to_op is None or not isinstance(from_op, to_op)
```

Comment:
yes, but why is there a special check for it here? It seems like `if to_op is None` is unnecessary, i.e. `None` would have been included in the comprehension in any case.